### ISSUE TITLE
[Adaptive Cards - Home Page] - Focus order is not logical on the page on after activating 'Close (X)' icon control present inside the video player dialog on the page.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9320]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)

**Repro Steps:**

1. Open the above URL on the Edge Dev browser.
2. Navigate to the 'Watch Video' control present below the 'A whole new way to deliver UI' heading text using tab key and activate using enter key.
3. Navigate to the 'Close (X)' icon control using tab key and activate using enter key.
4. Verify that, focus order is logical on the page on after activating 'Close (X)' icon control present inside the video player dialog on the page or not.

**Actual Result:**
Focus order is not logical on the page on after activating 'Close (X)' icon control present inside the video player dialog on the page. Currently on after activating the control focus gets lost and on pressing next Tab key press focus lands on the ' Adaptive Card Documentation Hub' control due to which users face difficulty on again navigating to the triggered control.

**Observation:**
Currently user have to press 9 times Tab keystrokes to reach the desired control and while in backward navigation user have to press 99 times Shift + Tab keystrokes.

**Expected Result:**
Focus order should be logical on the page on after activating 'Close (X)' icon control present inside the video player dialog on the page. Currently on after activating the control focus should not get lost and it should directly return back to the same triggered interactive control i.e., 'Watch Video' control.

**User Experience:**
Keyboard dependent users will be impacted if the focus order is not logical on the page on after activating 'Close (X)' icon control present inside the video player dialog on the page. Due to which users will not be able to perform the required action as it initiates the incorrect flow and user again need to perform extra keystrokes navigate to the required control and might get confused in terms of identifying the focus order of the page.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/focus-order.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/63753ea3-7aab-4ab4-90eb-6620dc71e58c)

https://github.com/user-attachments/assets/9f0967a4-1b8d-4852-aa25-c24e0f9af07a

### Card JSON

```JSON
NA
```
